redundant-bmc/src/meson.build

```diff
@@ -0,0 +1,13 @@
+installdir = join_paths(get_option('libexecdir'), 'phosphor-state-manager')
```

Comment:
i'll go research that.

Reply:
will leave at libexecdir to keep consistent with https://github.com/openbmc/phosphor-state-manager/blob/master/meson.build#L265

Reply:
Hmm ok, I’d prefer to rename it to `libexecinstalldir` since `installdir` is more generic. However, we can always rename it later if any conflicts arise

Reply:
oh, sorry, i though you were referring to the this one -> get_option('libexecdir').

I don't have an issue with updating the var name.

Reply:
Fixed.